docs/source/staticsite/configuration.rst

```diff
@@ -366,6 +366,19 @@ Parameters
     parameters
       staticsite_user_pool_arn: arn:aws:cognito-idp:<region>:<account-id>:userpool/<pool>
 
+**staticsite_additional_redirect_domains (Optional[str])**
```

Comment:
I would rather have this defined as a `List[str]` instead of `Optional[str]` (comma delimited list) to make the yaml configuration easier to read/write. I don't see a need for this to be a string when you are splitting it into a list on the backend anyway.

Also since the driving forces behind this is local development, it may be worth noting that use case here (adding the _why_ to your example).

Reply:
Great thought. I'd like to leave it as a `str` for now to match the aliases option, but would be open to changing both in the future. I realize it's a bit of a leaky abstraction (from CloudFormation) to have them be strings in the runway yaml. Shouldn't be too rough to support them either way (`if isinstance(str) then split)`) .

Somewhat related: at some point I'm probably going to PR and update to the CFN templates to process the values as CFNCommaDelimitedList Parameters (strings) again. Having used the pure-template versions now, I find that I miss the ChangeSet output when updating Parameters. I don't want to say that it's _vastly_ better, but it's noticeable.